test/integration/rules/aria-roledescription/aria-roledescription.html

```diff
@@ -15,6 +15,11 @@
 ></div>
 <input type="checkbox" aria-roledescription="my checkbox" id="pass8" />
 <input type="radio" aria-roledescription="my radio" id="pass9" />
+<div role="grid">
+  <div role="gridrow">
```

Comment:
`gridrow` is not a valid role.

```suggestion
  <div role="row">
```